app/src/main/java/org/p2p/wallet/home/events/MetadataLoader.kt

```diff
@@ -0,0 +1,14 @@
+package org.p2p.wallet.home.events
+
+import org.p2p.wallet.auth.interactor.MetadataInteractor
+
+class MetadataLoader(
+    private val metadataInteractor: MetadataInteractor
+) : HomeScreenLoader {
+
+    override suspend fun onLoad() {
+        metadataInteractor.tryLoadAndSaveMetadata()
```

Comment:
Is it OnboardingMetadata? 
Let's rename then. `MetadataLoader` -> `OnboardingMetadataLoader`